jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/EmailDeadlinesBaseTest.java

```diff
@@ -141,22 +144,80 @@ public void testDelayedEmailNotificationOnDeadline() throws Exception {
         assertTrue(list.contains("new-email@domain.com"));
         assertTrue(list.contains("darth@domain.com"));
 
-        MimeMessage msg = ((WiserMessage) getWiser().getMessages().get(0)).getMimeMessage();
+        MimeMessage msg = getWiser().getMessages().get(0).getMimeMessage();
         assertEquals(myBody, msg.getContent());
         assertEquals(mySubject, msg.getSubject());
         assertEquals("from@domain.com", ((InternetAddress) msg.getFrom()[0]).getAddress());
         assertEquals("replyTo@domain.com", ((InternetAddress) msg.getReplyTo()[0]).getAddress());
 
         final Address[] recipients = msg.getRecipients(RecipientType.TO);
         assertNotNull(recipients);
-        assertTrue(recipients.length == 2);
+        assertEquals(2, recipients.length);
         list = new ArrayList<String>(2);
         list.add(((InternetAddress) recipients[0]).getAddress());
         list.add(((InternetAddress) recipients[1]).getAddress());
         assertTrue(list.contains("new-email@domain.com"));
         assertTrue(list.contains("darth@domain.com"));
     }
     
+    
+    @Test(timeout = 10000)
+    public void testDelayedEmail2NotificationsOnDeadline() throws Exception {  
```

Comment:
doesn't throw any checked exception
```suggestion
    public void testDelayedEmail2NotificationsOnDeadline() {  
```